### PR TITLE
fix: fixes oauth token expired error

### PR DESCRIPTION
**Pull request by a-rampalli**
## What
Fixes syncs failing due to oauth token expiring mid sync

## How
https://linear.app/rudderstack/issue/ETL-260/hubspot-source-syncs-failing-with-the-oauth-token-used-to-make-this

### CODE REVIEW

File: airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py [hunk 1]
```diff
@@ -369,7 +375,7 @@ def get_json_schema(self) -> Mapping[str, Any]:
             json_schema["properties"]["properties"] = {"type": "object", "properties": self.properties}
         return json_schema
 
-    @retry_connection_handler(max_tries=2)
+    @retry_connection_handler(max_tries=5)
```
Comment (nidhilashkari17) [inline airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py:378]: @a-rampalli why are we increasing the count of retries here ?

**Review by nidhilashkari17 — APPROVED** [review]
LGTM apart from one clarification comment.

File: airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py (hunk 1 above)
Comment (a-rampalli) [reply airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py:378]: Airbyte is retrying this error 5 times and in general all retries we are doing are either 3 or 5. Just this nothing specific

Comment (a-rampalli) [reply airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py:378]: Resolving as PR merge not allowed with open comments

**Pull request merged**